tools/run_tests.sh

```diff
@@ -65,6 +69,10 @@ for arg in "$@"; do
         ASAN_ONLY=true
         TEST_SCOPE="no_timing_test"
         ;;
+    "tsan-only")
+        CONTINUE_ON_ERROR=true
```

Comment:
Can this be removed? If I understood this correctly then the CI would hide detected errors, right? This should be done explicitly by the `continue-on-error` flag

Reply:
@elBoberido You're right. Done!